### PR TITLE
Extend <Divider /> skins with additional values

### PR DESCRIPTION
**Pull request by rudnitskih**
### 🔦 Summary

Add additional skins to support embedding into `<SectionHelper />`. [Design spec](https://app.abstract.com/projects/d56fb180-16ec-11e8-88a3-6b21f882bfc8/branches/6a09deac-f15f-4912-b9c1-1be8b880abe6/commits/latest/files/4411e21e-11af-4a25-8c4c-3aa7fc7136d0/layers/DA470827-C2F4-4831-9C86-A84B7356513B?collectionId=5def9524-0220-4ada-b440-991ef35c7b85&collectionLayerId=637a60d0-cb93-41bf-9edb-be74d1ff606e) 

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- [x] 👨‍🎨 UX change is approved by the Design System UX @milkyfruit 
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [X] Visual test

### CODE REVIEW

File: src/Divider/Divider.js [hunk 1]
```diff
@@ -14,7 +14,15 @@ class Divider extends React.PureComponent {
     /** Indicates whether to display the divider horizontally or vertically */
     direction: PropTypes.oneOf(['horizontal', 'vertical']),
     /** Sets the color of the divider */
-    skin: PropTypes.oneOf(['light', 'dark']),
+    skin: PropTypes.oneOf([
+      'light',
+      'dark',
+      'standard',
```
Comment (rudnitskih) [inline src/Divider/Divider.js:20]: @Domantas-wix I think `standard` is a confusing name for this case.
Maybe should we find something different?

File: src/Divider/docs/index.story.js [hunk 2]
```diff
@@ -68,9 +67,9 @@ export default {
           }),
 
           example({
-            title: 'Example with Dark Skin',
+            title: 'Example with different skin',
             text:
-              'The divider could be displayed in dark color using the `skin` prop.',
+              'The divider could be displayed in different color using the `skin` prop.',
```
Comment (rudnitskih) [inline src/Divider/docs/index.story.js:72]: @milkyfruit please review this content

**Comment by rudnitskih** [comment]
> Looks good to me
> I think we can replace "skin" example code with this:
> 
> ```
> <Layout>
>   <Cell>
>     <Divider skin="light" />
>   </Cell>
>   <Cell>
>     <Divider skin="dark" />
>   </Cell>
>   <Cell>
>     <Divider skin="standard" />
>   </Cell>
>   <Cell>
>     <Divider skin="warning" />
>   </Cell>
>   <Cell>
>     <Divider skin="destructive" />
>   </Cell>
>   <Cell>
>     <Divider skin="success" />
>   </Cell>
>   <Cell>
>     <Divider skin="premium" />
>   </Cell>
> </Layout>
> ```
> 
> @milkyfruit what do you say?

@EgoziE I was inspired by this example -- https://github.com/wix/wix-style-react/blob/master/src/SectionHelper/test/SectionHelper.visual.js

And I personally like it, it simplifies maintenance.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5694/

**Comment by EgoziE** [comment]
> > Looks good to me
> > I think we can replace "skin" example code with this:
> > ```
> > <Layout>
> >   <Cell>
> >     <Divider skin="light" />
> >   </Cell>
> >   <Cell>
> >     <Divider skin="dark" />
> >   </Cell>
> >   <Cell>
> >     <Divider skin="standard" />
> >   </Cell>
> >   <Cell>
> >     <Divider skin="warning" />
> >   </Cell>
> >   <Cell>
> >     <Divider skin="destructive" />
> >   </Cell>
> >   <Cell>
> >     <Divider skin="success" />
> >   </Cell>
> >   <Cell>
> >     <Divider skin="premium" />
> >   </Cell>
> > </Layout>
> > ```
> > 
> > 
> > @milkyfruit what do you say?
> 
> @EgoziE I was inspired by this example -- https://github.com/wix/wix-style-react/blob/master/src/SectionHelper/test/SectionHelper.visual.js
> 
> And I personally like it, it simplifies maintenance.

There's a difference between a visual test code to an example code.
I agree that visual test code should be clean and cover everything.
But example code should be "ready to copy paste" :)

File: src/Divider/Divider.js (hunk 1 above)
Comment (milkyfruit) [reply src/Divider/Divider.js:20]: Please keep it as "standard". It's officially defined name in WSR that's used for components in blue color – buttons, section helpers, notificitoans, badges and more.

File: src/Divider/docs/index.story.js (hunk 2 above)
Comment (milkyfruit) [reply src/Divider/docs/index.story.js:72]: Looks fine for this PR.

**Comment by rudnitskih** [comment]
@EgoziE I didn't change an example for this task:

image

Consumers can see all available skins in Playground and API tabs:

image
image

**Pull request merged**